doc/architecture.rst

```diff
@@ -103,17 +97,18 @@ forth.
 Scalability and High Availability
 ---------------------------------
 
-In traditional architectures, clients talk to a centralized component (e.g., a
-gateway, broker, API, facade, etc.), which acts as a single point of entry to a
-complex subsystem. This imposes a limit to both performance and scalability,
-while introducing a single point of failure (i.e., if the centralized component
-goes down, the whole system goes down, too).
+In traditional architectures, clients talk to a centralized component. This
+centralized component might be a gateway, a broker, an API, or a facade. A
+centralized component of this kind acts as a single point of entry to a complex
+subsystem. Architectures that rely upon this kind of centralized component
+incur a limit to performance and scalability and have a single point of
+failure. If the centralized component goes down, the whole system goes down.
```

Comment:
the whole system is unavailable.

Reply:
That is better.

Reply:
I've gone with "becomes unavailable" to emphasize the procedural cause-and-effect nature of the unavailability of the central component.